### PR TITLE
[WWST-4210] Add LEDVance 15" Adjustable White Edge LitFlush Mount to zigbee-dimmer

### PR DESCRIPTION
**Pull request by MGoralczykS**
@greens @MMateusiakS @ZWozniakS @KKlimczukS 
Could you review this? 

RAW -> 03 0104 0102 02 08 0000 0003 0004 0005 0006 0008 0300 FC0F 01 0019

### CODE REVIEW

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 1]
```diff
@@ -64,6 +64,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0019", manufacturer: "Aurora", model: "WallDimmerMaster", deviceJoinName: "Aurora Smart Rotary Dimmer"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E1A-AC2", deviceJoinName: "Sengled DownLight"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "MLI", model: "ZBT-DimmableLight", deviceJoinName: "Müller Licht Tint Bulb Dimming"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FC0F", outClusters: "0019", manufacturer: "OSRAM", model: " LIGHTIFY Edge-lit flushmount", deviceJoinName: "LEDVance 15\" Adjustable White Edge LitFlush Mount"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: This join name is what is on the ticket but I don't think it's correct. I asked for clarification on the ticket so let's not merge this until we confirm the join name.

Comment (MGoralczykS) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: @tpmanley Any update here? :)

Comment (greens) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: there are conflicts now and there's a space before the model name that probably should not be there

Comment (tpmanley) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Per the ticket please change the join name to "SYLVANIA SMART+ Flush Mount"

Comment (MGoralczykS) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Done :)

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 2]
```diff
@@ -64,6 +64,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0019", manufacturer: "Aurora", model: "WallDimmerMaster", deviceJoinName: "Aurora Smart Rotary Dimmer"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E1A-AC2", deviceJoinName: "Sengled DownLight"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "MLI", model: "ZBT-DimmableLight", deviceJoinName: "Müller Licht Tint Bulb Dimming"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FC0F", outClusters: "0019", manufacturer: "OSRAM", model: " LIGHTIFY Edge-lit flushmount", deviceJoinName: "SYLVANIA SMART+ Flush Mount"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Is there supposed to be a space at the beginning of the model name?

**Review by tpmanley — DISMISSED** [review]

**Review by greens — DISMISSED** [review]

**Review by greens — APPROVED** [review]

**Pull request merged**